docs/api-spec-swagger.yaml

```diff
@@ -0,0 +1,3370 @@
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Job approved successfully
+        '401':
+          description: Unauthorized - Invalid token
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Unauthorized - Invalid token
+                  
+        '403':
+          description: Forbidden - User does not have permission
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Forbidden - User does not have permission
+                  
+        '404':
+          description: Job not found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Job not found
+  /jobs/{jobId}/reject:
+    post:
+      tags:
+        - job
+      summary: Reject a new job submission. 
+      description: Must have ADMIN authority.
+      operationId: rejectJob
+      security:
+        - jwt: []
+      parameters:
+        - name: jobId
+          in: path
+          description: Id of job to reject
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: Job rejected successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Job approved successfully
+        '401':
+          description: Unauthorized - Invalid token
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Unauthorized - Invalid token
+                  
+        '403':
+          description: Forbidden - User does not have permission
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Forbidden - User does not have permission
+                
+        '404':
+          description: Job not found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: Job not found
+  /jobs/{jobId}/save:

```

Comment:
should be in jobs section. also /jobs/{jobId}/save/{userId} without request body